### PR TITLE
[IMPAC-534] Add currency info to sales widgets

### PR DESCRIPTION
**Pull request by cesar-tonnoir**
- Added currency info to invoices list widget

### CODE REVIEW

**Comment by cesar-tonnoir** [comment]
Not ready - will do the same to the other sales widgets
invoices-list

**Comment by cesar-tonnoir** [comment]
@xaun ready to be reviewed.

**Comment by cesar-tonnoir** [comment]
@xaun need your review please

File: src/components/widgets/sales-comparison/sales-comparison.directive.coffee [hunk 1]
```diff
@@ -207,6 +210,32 @@ module.controller('WidgetSalesComparisonCtrl', ($scope, $q, $filter, ChartFormat
...
 
+  buildFxTotals = ->
+    for groupedSales in w.content.sales_comparison
+      for sale in groupedSales.sales
+        sale.formattedFxTotals = {}
+        netSaleFxTotals = []
+        grossSaleFxTotals = []
+        unless _.isEmpty(sale.fx_totals)
+          _.mapKeys sale.fx_totals, (total, currency) ->
+            for grossAmount in total['net_value_sold']
```
Comment (xaun) [inline src/components/widgets/sales-comparison/sales-comparison.directive.coffee]: `total['net_value_sold']` Ruby syntax spill-over?

Comment (cesar-tonnoir) [inline src/components/widgets/sales-comparison/sales-comparison.directive.coffee]: yeah, it's part of the API response, that's why

**Pull request merged**